pep-0594.rst

```diff
@@ -167,33 +132,32 @@ audio processing.
    :header: "Module", "Deprecated in", "To be removed", "Added in", "Has maintainer?", "Replacement"
    :widths: 2, 1, 1, 1, 1, 2
 
-    aifc,3.8 (3.0\*),3.10,1993,**yes (inactive)**,\-
+    aifc,3.11 (3.0\*),3.13,1993,**yes (inactive)**,\-
     asynchat,**3.6** (3.0\*),3.10,1999,**yes**,asyncio_
     asyncore,**3.6** (3.0\*),3.10,1999,**yes**,asyncio_
```

Comment:
```suggestion
    asynchat,**3.6** (3.0\*),3.12,1999,**yes**,asyncio_
    asyncore,**3.6** (3.0\*),3.12,1999,**yes**,asyncio_
```

These modules (and `smtpd`) had formal `DeprecationWarnings` added in 3.10 per [bpo-44498](https://bugs.python.org/issue44498), were attempted to be removed in 3.11 but the change was reverted after you refereed it to the SC in python/steering-council#86 and a PEP 387 exception was rejected, so per policy this will now land in 3.12. See [bpo-28533](https://bugs.python.org/issue28533) for others curious.